.gitlab/deploy_containers/deploy_containers_a7.yml

```
@@ -44,6 +44,27 @@ include:
     - export IMG_SOURCES="${IMG_BASE_SRC}-7${JMX}-win${OS}${FLAVOR}-amd64"
     - export IMG_DESTINATIONS="${AGENT_REPOSITORY}:${VERSION}${FLAVOR}-ltsc${LTSC_VERSION}${JMX}"
 
+.deploy_containers-a7-base-fips:
+  extends: .docker_publish_job_definition
+  stage: deploy_containers
+  dependencies: []
+  before_script:
+    - if [[ "$VERSION" == "" ]]; then VERSION="$(dda inv agent.version --url-safe --pipeline-id $PARENT_PIELINE_ID)-fips" || exit $?; fi
```

🥜 and sorry as you just moved this code
```suggestion
    - if [[ -z "$VERSION" ]]; then VERSION="$(dda inv agent.version --url-safe --pipeline-id $PARENT_PIELINE_ID)-fips" || exit $?; fi
```